cmd/crictl/exec.go

```diff
@@ -427,7 +427,28 @@ func stream(ctx context.Context, in, tty bool, transport string, parsedURL *url.
 		return errors.New("input is not a terminal")
 	}
 
-	streamOptions.TerminalSizeQueue = t.MonitorSize(t.GetSize())
+	streamOptions.TerminalSizeQueue = &terminalSizeQueueAdapter{queue: t.MonitorSize(t.GetSize())}
 
 	return t.Safe(func() error { return executor.StreamWithContext(ctx, streamOptions) })
 }
+
+// terminalSizeQueueAdapter adapts k8s.io/kubectl/pkg/util/term.TerminalSizeQueue
+// to k8s.io/client-go/tools/remotecommand.TerminalSizeQueue.
+// TODO: Remove this adapter once kubernetes/kubernetes#135587 is merged and vendored.
```

Comment:
interesting. I wonder if there will be many complaints about it so we can justify cherry-pick to 1.35